### ISSUE TITLE
Hide code spew on abort of ssh login

### ISSUE THREAD

**Issue by frederickjh** [xxh/xxh#72]
**Is your feature request related to a problem? Please describe.**
Some times I do not have the ssh key in my ssh agent when I  go to xxh into a host. I am then asked for my password but since want to use my key I abort with **Ctrl+C**. Then I see this:

```bash
$ xxh remotehost +if +RI xxh-plugin-fish-userconfig+path+/home/localusername/workarea/xxh-plugin-fish-userconfig
Remove xxh-plugin-fish-userconfig
Removed /home/localusername/.xxh/.xxh/plugins/xxh-plugin-fish-userconfig
Install xxh-plugin-fish-userconfig+path+/home/localusername/workarea/xxh-plugin-fish-userconfig to local /home/localusername/.xxh/.xxh/plugins/xxh-plugin-fish-userconfig
Package source path: /home/localusername/workarea/xxh-plugin-fish-userconfig
Build xxh-plugin-fish-userconfig
Installed /home/localusername/.xxh/.xxh/plugins/xxh-plugin-fish-userconfig
remoteusername@remotehost's password: 


Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/xxh", line 14, in <module>
    xxh.main()
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/xxh_xxh/xxh.py", line 704, in main
    host_info = self.get_host_info()
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/xxh_xxh/xxh.py", line 268, in get_host_info
    pr = self.pssh(cmd)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/xxh_xxh/xxh.py", line 146, in pssh
    user_host_password = getpass.getpass(prompt=(sess.before + sess.after).decode("utf-8")+' ')
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/getpass.py", line 77, in unix_getpass
    passwd = _raw_input(prompt, stream, input=input)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/getpass.py", line 146, in _raw_input
    line = input.readline()
KeyboardInterrupt
```

**Describe the solution you'd like**
Not sure if this is possible but possibly a clean exit without all this debug info. Maybe a debug option could be set to dump this information if needed.

**Describe alternatives you've considered**
Live with it as it is.

**Additional context**
None
**For community:**
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by frederickjh** [comment]
[Python Catch SIGINT (CTRL-C)](https://www.devdungeon.com/content/python-catch-sigint-ctrl-c)

**Comment by anki-code** [comment]
Will be released in 0.7.9
